gallery/src/GalleryNav/GalleryNav.tsx

```diff
@@ -21,43 +21,77 @@ import {
   complement,
   values
 } from 'ramda';
-import { NxTreeView, NxTreeViewChild, NxFilterInput, NxTreeViewProps } from '@sonatype/react-shared-components';
+import {
+  NxCollapsibleItems,
+  NxFilterInput,
+  NxCollapsibleItemsProps,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
 
-import pageConfig from './pageConfig';
+import pageConfig from '../pageConfig';
 import { useLocation } from 'react-router';
 import { NavLink } from 'react-router-dom';
-import { PageMapping, PageConfig } from './pageConfigTypes';
-import { markByFilter, matchesFilter } from './filterUtil';
-
-const renderLinks = (filter?: string) => pipe(
-    keys,
-    map((pageName: string) =>
-      <NxTreeViewChild key={pageName}>
-        <NavLink to={`/pages/${pageName}`} activeClassName="selected">
-          {markByFilter(filter, pageName)}
-        </NavLink>
-      </NxTreeViewChild>
-    )
-);
+import { PageMapping, PageConfig, PageType, PageContentDescription } from '../pageConfigTypes';
+import { markByFilter, matchesFilter } from '../filterUtil';
+
+import { faFile, faCode, faRulerCombined } from '@fortawesome/free-solid-svg-icons';
+import { faReact, faSass, faJs } from '@fortawesome/free-brands-svg-icons';
```

Comment:
This is the warning that FA displays in their documentation for each of the "Brands" icons:

> This work includes material that may be protected as a trademark in some jurisdictions. If you want to use it, you have to ensure that you have the legal right to do so and that you do not infringe any trademark rights. The use of these trademarks does not indicate endorsement of the trademark holder by Font Awesome, nor vice versa. See the trademark owner for rules about appropriate use of their trademarks.

Reply:
> This is the warning that FA displays in their documentation for each of the "Brands" icons:
> 
> > This work includes material that may be protected as a trademark in some jurisdictions. If you want to use it, you have to ensure that you have the legal right to do so and that you do not infringe any trademark rights. The use of these trademarks does not indicate endorsement of the trademark holder by Font Awesome, nor vice versa. See the trademark owner for rules about appropriate use of their trademarks.

I have a extension for VS code that shows very very similar icons for all of the file types and I imagine M$ would be pretty on top of that if there were a problem so I think we're safe.

Reply:
> I have a extension for VS code that shows very very similar icons for all of the file types and I imagine M$ would be pretty on top of that if there were a problem so I think we're safe.

JS logo comes from here afaict: https://github.com/voodootikigod/logo.js, MIT licensed
SASS: https://sass-lang.com/styleguide/brand, Creative Commons license, they have requests about color though.
React: creative commons licensed

All of these require attribution I believe.  Not sure if that's handled implicitly through fontawesome or not.

Reply:
> Not sure if that's handled implicitly through fontawesome or not.

Based on https://github.com/FortAwesome/Font-Awesome/blob/master/js-packages/%40fortawesome/free-brands-svg-icons/LICENSE.txt#L20, it seems that we shouldn't have to worry about it.